### PR TITLE
add a flag to the metadata for thunderbird donations

### PR DESCRIPTION
**Pull request by cadecairos**
For #1368

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -76,6 +76,10 @@ var routes = {
     };
     var request_id = request.headers['x-request-id'];
 
+    if (transaction.description === "Thunderbird") {
+      metadata.thunderbird = true;
+    }
```
Comment (ScottDowne) [inline routes/index.js]: Is there a reason you need it as a flag and not a desc? Or you just feel this is better overall?

Comment (ScottDowne) [inline routes/index.js]: accounting relies on this param being description and not a flag. I'm not sure how that process works, though. Feels busted to me.

Comment (cadecairos) [inline routes/index.js]: Does accounting take into account monthly subscriptions, where charges don't have a description set?

This method that uses metadata ensures that everytime a charge comes through, this flag is on the customer object, even for monthly subs

Comment (ScottDowne) [inline routes/index.js]: But does it need to be a boolean?

Comment (cadecairos) [inline routes/index.js]: No, but description remains untouched as "Thunderbird". This flag is for the donation email rendering code

Comment (ScottDowne) [inline routes/index.js]: OK, that works for me. I wonder why we didn't try that in the first place for description?

So this makes me wonder if accounting knows how to get thunderbird subscriptions at all? My guess is no.

Maybe we're not passing description at all for subscriptions and we'll need to pass it into metadata anyway. Thus, we would end up with description and the boolean, which, if you're fine with, so I am?

**Comment by ScottDowne** [comment]
@cadecairos yeah, see this line https://github.com/mozilla/donate.mozilla.org/blob/master/mixins/form.jsx#L311

That changes the description.

**Pull request merged**